src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -76,15 +77,25 @@
      (capture-reports* reports# (fn [] ~@body))
      @reports#))
 
+(defn times [options]
+  (cond (map? options)     (:num-tests options tc.clojure-test/*default-test-count*)
+        (integer? options) options))
+
+(defn other-options [options]
+  (cond (map? options)     (dissoc options :num-tests)
+        (integer? options) {}))
+
 (defmacro qc-and-report-exception
-  [final-reports tests bindings & body]
+  [final-reports options bindings & body]
```

Comment:
Actually, if it's OK with you, I'll change it to `num-tests-or-options` throughout - it's a little more verbose but I like the explicitness.